company/okrs/2020_q1.md

```diff
@@ -103,3 +102,13 @@ Build out vision for product and run customer validation discussions
 
 [N]: https://docs.google.com/document/d/1yndPaKSiB4Jq6J6cwGzcIUBSIupPOySHUHsOF2ipqFo/edit#bookmark=kix.n8t17z6iyawc
 [M]: https://docs.google.com/document/d/1yndPaKSiB4Jq6J6cwGzcIUBSIupPOySHUHsOF2ipqFo/edit#bookmark=id.vwn1af52n0ns
+
+## Operations
+
+1. **VP Business: Make Sourcegraph a lean organization that uses data and budgets to guide decision making.**
+   1. VP Business: Provide a board-approved budget (including spending and hiring plans) to each department head.
+   1. VP Business: Provide dashboards to each department head that are used on at least a weekly basis (prioritizing [sales](https://sourcegraph.looker.com/dashboards/123), [marketing](https://sourcegraph.looker.com/dashboards/118), and customer success).
+
+1. **VP Business: Build a rewarding, challenging, and inclusive people-ops operating system.**
+   1. VP Business: Build the HR and legal policies that will enable an all-remote team to operate smoothly (e.g., ability to contract or be employed in all necessary locations, smooth national and cross-border payments, fair benefits for all teammates, a professional development story for each function, and a process for everyone to raise questions or concerns and to get answers).
+   1. VP Business: Write the handbook documentation and implement recruiting and onboarding practices, teammate trainings, and more, to help ensure candidates and teammates from every community feel included.
```

Comment:
This one IS subjective, so my alternative is to rephrase it around coming to agreement on what we will actually be included in this effort (since there's still a bit of ambiguity). Happy to discuss this one more in detail.